### DISCUSSION TITLE
短影音潮流：我們該何去何從？

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1705] [General]
前陣子看到一個很有趣的主題：短影音

現在shorts是流量密碼，不論是YouTube、Facebook、Instagram，或是中國的抖音、小紅書，豐富且複雜的資訊滿佈在這些平台、充斥於我們的日常，那這究竟是好是壞呢？

有人說：
現代人的時間都很瑣碎，可以利用短短的幾十秒汲取新知，了解天下事；或者看到博君一笑的內容能夠放鬆心情，短暫卸下疲憊。

也有人說：
如此複雜的資訊充斥在我們四周，來去非常迅速，會讓大腦的專注力下降；或是一個不注意，大把的時間就被浪費掉了。

我覺得其實就是一把雙面刃，這些優缺點都是客觀存在的事實，那我們該如何揚長避短呢？我認為選擇和自制力是兩大關鍵。選擇看適合的短影音，而不是來者不拒，讓沒內涵的東西進入大腦；觀看的時間則是依靠自制力去管控，不讓自己沉迷在影音世界，浪費大好時光。

**Comment by 12Jesse24** [comment]
確實現在很多shorts都是流行的趨勢，很多朋友們也常常在聊天時使用其中的梗，但花太多時間去觀看反而會造成反效果，就像樓主說的:
觀看的時間是依靠自制力去管控，因此我覺得短影音帶來的好壞取決於個人。

**Comment by THH1118** [comment]
自制力跟面對很多資訊時的識讀能力真的很重要，但這種短影音就是會設計的讓人很想一部一部一直看下去，自己就是會不小心看太久的那群人，有時候明明知道有重複但還是會再看一次好可怕。

**Comment by davidteng00000** [comment]
短影音確實黑容易上癮，一直滑一直滑就可以獲得足夠的娛樂，這對於一個很忙的人來說真的是一件非常紓壓的事情，畢竟日常生活中可能沒有時間與心力去安排好的娛樂項目。我們也只能盡力避免上癮短影音，多從事一些運動

**Comment by Patsy17** [comment]
現在也有許多時事短影音可以讓我們節省時間去觀看，但是觀看者還是需要有自主辨別內容真實性的能力。

**Comment by asd99263fg** [comment]
真的這個就跟毒品一樣，很容易看上癮，所以真的不能看太多，不然就跟吸毒一樣，短影音很容易讓人注意力降低，現在看一堆之後都覺得要好好專注一件事情變成好難，真的好可怕

**Comment by Piyenparty** [comment]
我覺得這就是一種時代的潮流，演算法統計大數據然後根據人民的喜好發放影音內容，即使今天是長影音，對於沒有克制力的使用者們(像我)而言，一樣會起到吸引使用者大量使用的作用。時代更迭沒有不好，影音文化在進步，我們也應該培養自己的克制力

**Comment by RU10171017** [comment]
我覺得你說得很中肯，短影音真的像是一把雙面刃。我自己有時候也會刷 shorts 或 reels，一開始是想放鬆一下，結果常常不小心就花了好幾十分鐘，有點像是被節奏「帶著走」。但換個角度想，它也真的讓很多知識或觀點更容易被接觸到，像我就是透過短影音認識一些以前完全沒接觸過的議題或技能。我覺得你提到「選擇」跟「自制力」真的很重要，現在的資訊太多了，主動選擇什麼進到自己腦袋裡，比以前重要得多。不然真的一不小心，就變成資訊的「容器」，不是主人了。

**Comment by LINSHIHMIN** [comment]
短影音確實可以讓我們更快速地接受到想要的資訊，如果是迷因之類的好笑的東西就算了，但如果是知識型的短影音，會為了不超過時間而大幅縮短該講解的內容，有些可能是精簡再精簡的內容，有些影片則是會直接斷章取義，或是說明得不清不楚讓人產生誤會而造成謠言。還有另一個問題是我最近發現那些shorts reels明明已經是很短的內容了，但現在甚至還可以兩倍速觀看，現在人的專注力感覺又再更下滑。當然也不是說短影片有多麼不好，但至少一天中找個其他事情來做，不要讓自己的專注力變得太差，比如說看個一兩小時的書也好，在短影片盛行的時代，我們除了接受以外也要懂得克制自己，而不是把時間全部浪費掉。

**Comment by wuyu06** [comment]
短影音確實改變了我們獲取資訊和娛樂的方式，方便又快速，但也容易讓人分心甚至浪費時間。這篇點出關鍵：選擇和自制力。要有意識地挑選有價值的內容，避免無腦刷屏，才能讓短影音成為助力而非負擔

**Comment by edawdewv** [comment]
我自己其實也曾陷入短影音的漩渦，原本只是想看一兩支影片放鬆，結果一滑就一兩個小時過去了。那時候覺得蠻挫敗的，因為本來安排好的進度完全被打亂。後來我開始設定手機的螢幕時間限制，也會在看短影音前先問自己：「我現在是真的需要放鬆，還是只是想逃避別的事情？」
但我也有因為短影音接觸到一些原本不會主動搜尋的知識，例如歷史故事的整理、冷知識、甚至是一些程式教學的碎片內容，反而激起了我深入了解的動機。
